framework/base/Controller.php

```diff
@@ -449,7 +452,21 @@ public function setView($view)
      */
     public function getViewPath()
     {
-        return $this->module->getViewPath() . DIRECTORY_SEPARATOR . $this->id;
+        if ($this->_viewPath !== null) {
+            return $this->_viewPath;
+        } else {
+            return $this->_viewPath = $this->module->getViewPath() . DIRECTORY_SEPARATOR . $this->id;
+        }
```

Comment:
I agree, but I copied the code block from `yii\base\Module` to make sure I was keeping consistent style. Should I change?